fcp-0000.md

```diff
@@ -73,7 +73,7 @@ An FCP SHOULD address the following questions:
 The problem should be clearly described in sufficient detail such that someone
 unfamiliar with the issue can gain enough context to understand the proposed
 solution. The description of the problem SHOULD NOT be combined with the
-description of the solution. Ideally the problem is described first and
+description of the solution. Ideally, the problem is described first and
 discussed followed by a description of the intended solution(s) and discussion
```

Comment:
How about just writing "The problem is..." and leaving out the "Ideally" altogether?